docker/README.md

```diff
@@ -72,15 +72,15 @@ sudo docker run -ti -e TERM=linux --name gitlab-ce-troubleshoot --publish 8080:8
 ### Get published images from Dockerhub
 
 ```bash
-sudo docker pull sytse/gitlab-data
+sudo docker pull sytse/gitlab_data
```

Comment:
Then you have to rename the image at docker-hub. A docker pull won't work with `sytse/gitlab-data` at the moment.